UX principles.md

```diff
@@ -0,0 +1,40 @@
+# User experience principles
+
+Channel UX principles are inspired by [Shopify Values](https://polaris.shopify.com/foundations/experience-values "Shopify values") and are at the core of how we build our channels.
```

Comment:
```suggestion
Channel UX principles are inspired by [Shopify experience values](https://polaris.shopify.com/foundations/experience-values "Shopify values") and are at the core of how we build our channels.
```

Not clear on the convention of putting quoted text in the link. Is it a GH-flavored markdown thing? If it's kept in, should probably be Shopify experience values, as that's what's used in the referenced text.